src/design-system/globals/globalStyle.tsx

```diff
@@ -42,9 +42,13 @@ export const GlobalStyle = createGlobalStyle<{ theme: DefaultTheme }>`
 
 	h1 {
 		font-size: ${({ theme }) => theme.fontSizes.h1};
+		padding: 2.4rem 0 ${({ theme }) => theme.spaces.xxs};
 	}
 	h2 {
-		font-size: ${({ theme }) => theme.fontSizes.h2};
+		font-size: ${({ theme }) => theme.fontSizes.s};
```

Comment:
You probably should change the value of the `fontSizes.h2` field and use it for our h2 :).

Reply:
I changed `fontSizes.h2` and `fontSizes.h3` otherwise this last would have been bigger than the first.
I actually gave `fontSizes.h2` the previous value of `fontSizes.h3`, and I gave `fontSizes.h3` the new value I wanted to give to `fontSizes.h2`, it's more consistent this way.
I reversed `<h2 />` and `<h3 />` tags in the code base.
Nothing changes on the app.
cd8a90f